src/containers/familyDetails/index.tsx

```diff
@@ -0,0 +1,70 @@
+import { Alert, Spin } from 'antd';
+import React, { useEffect, useState } from 'react';
+import { Helmet } from 'react-helmet';
+import { useDispatch } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import protectedApiClient from '../../api/protectedApiClient';
+import { ChungusContentContainer } from '../../components';
+import ContactInfoSummary from '../../components/ContactInfoSummary';
+import {
+  AsyncRequest,
+  AsyncRequestCompleted,
+  AsyncRequestFailed,
+  asyncRequestIsComplete,
+  asyncRequestIsFailed,
+  asyncRequestIsLoading,
+  asyncRequestIsNotStarted,
+  AsyncRequestLoading,
+  AsyncRequestNotStarted,
+} from '../../utils/asyncRequest';
+import { ContactInfo } from '../setContacts/ducks/types';
+
+const FamilyDetails: React.FC = () => {
+  const dispatch = useDispatch();
+  const id = Number(useParams<{ id: string }>().id);
+  const [contacts, setContacts] = useState<AsyncRequest<ContactInfo, any>>(
```

Comment:
Is this an okay use of any?

Reply:
At this point we really don't have a good interface to work with. I suppose the easiest thing would have been to make it a string and just store the error message throughout the codebase. But we can convert this to use the utility later.